ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```diff
@@ -83,7 +83,8 @@ if [[ -n "${LOOKBACK_SIZE}" ]]; then
 fi
 
 set +e
-es_metadata_index=${ES_METADATA_INDEX} es_benchmark_index=${ES_BENCHMARK_INDEX} VERSION=${VERSION} orion cmd --config ${CONFIG} ${EXTRA_FLAGS}
+FILENAME=$(echo $CONFIG | awk -F/ '{print $2}' | awk -F. '{print $1}')
```

Comment:
The above logic might break for some corner cases where paths are lengthy. So we need the below change where `basename` will take care of everything for us.
```suggestion
FILENAME=$(basename "$CONFIG" | cut -d. -f1)
```